plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/model/DatabricksMetaModel.java

```diff
@@ -37,4 +52,38 @@ public DBCQueryTransformer createQueryTransformer(@NotNull DBCQueryTransformType
         }
         return null;
     }
+
+    @Override
+    public GenericDataSource createDataSourceImpl(DBRProgressMonitor monitor, DBPDataSourceContainer container) throws DBException {
+        return new DatabricksDataSource(monitor, container, this);
+    }
+
+    @Override
+    public String getTableDDL(DBRProgressMonitor monitor, GenericTableBase
+            sourceObject, Map<String, Object> options) throws DBException {
+        GenericDataSource dataSource = sourceObject.getDataSource();
+        try (JDBCSession session = DBUtils.openMetaSession(monitor, sourceObject, "Read Databricks view/table source")) {
+            try (JDBCPreparedStatement dbStat = session.prepareStatement(
+                    "SHOW CREATE TABLE " + sourceObject.getFullyQualifiedName(DBPEvaluationContext.DDL))) {
+                try (JDBCResultSet dbResult = dbStat.executeQuery()) {
+                    StringBuilder sql = new StringBuilder();
+                    while (dbResult.next()) {
+                        String line = dbResult.getString(1);
+                        if (line == null) {
+                            continue;
+                        }
+                        sql.append(line).append("\n");
+                    }
+                    String ddl = sql.toString();
+                    if (sourceObject.isView()) {
+                        ddl = ddl.replace("CREATE VIEW", "CREATE OR REPLACE VIEW");
```

Comment:
If you read view definitions as well here, you need to add

https://github.com/dbeaver/dbeaver/blob/469ccea7637e875a98b5f9abe6f8da2720e03334/plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseMetaModel.java#L162-L165

Reply:
@LonwoLonwo thanks, good catch. I missed it as the driver returns both views and tables with type `TABLE`.